src/materials/shaders/pointcloud.vs

```diff
@@ -490,6 +490,9 @@ vec3 getColor(){
 		color = getRGB();
 	#elif defined color_type_height
 		color = getElevation();
+	#elif defined color_type_rgb_highlight
+		// 124, 215, 244
+		color = vec3(0.486, 0.843, 0.957);
```

Comment:
@damian66 Might be worth changing to `color = vec3(124 / 255, 215 / 255, 244 / 255)` in case we want to quickly change this colour later?